ocf_data_sampler/torch_datasets/sample/uk_regional.py

```diff
@@ -58,14 +177,60 @@ def plot(self) -> None:
             axes[0, 0].set_title("GSP Generation")
 
         if "solar_azimuth" in self._data and "solar_elevation" in self._data:
-                    axes[1, 1].plot(self._data["solar_azimuth"], label="Azimuth")
-                    axes[1, 1].plot(self._data["solar_elevation"], label="Elevation")
-                    axes[1, 1].set_title("Solar Position")
-                    axes[1, 1].legend()
+            axes[1, 1].plot(self._data["solar_azimuth"], label="Azimuth")
+            axes[1, 1].plot(self._data["solar_elevation"], label="Elevation")
+            axes[1, 1].set_title("Solar Position")
+            axes[1, 1].legend()
 
         if SatelliteSampleKey.satellite_actual in self._data:
             axes[1, 0].imshow(self._data[SatelliteSampleKey.satellite_actual])
             axes[1, 0].set_title("Satellite Data")
 
         plt.tight_layout()
         plt.show()
+
+
+def validate_samples(
```

Comment:
I'd probably be tempted to strip this out for the moment. We should just be able to use the code above to validate.